kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/DefaultConfiguration.java

```diff
@@ -33,7 +34,19 @@
 public class DefaultConfiguration {
 
   // initialize the default configuration from the environment
-  private static Configuration conf = new Configuration();
+  private static Configuration conf;
+
+  static{
+    //read system property for Oozie Action Configuration XML
+    String oozieActionXml = System.getProperty("oozie.action.conf.xml");
+    if(oozieActionXml != null && oozieActionXml.length() > 0) {
+      Configuration.addDefaultResource(new Path("file:///", oozieActionXml).toString());
```

Comment:
Yes, I think they do, since default resources are not interpreted as Paths, but Strings: https://github.com/apache/hadoop/blob/trunk/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/conf/Configuration.java#L2605-L2619